spec/features/eventmanager_spec.rb

```diff
@@ -0,0 +1,78 @@
+require "rails_helper"
+require "database_cleaner"
+
+RSpec.feature "Event Manager abilities", type: :feature, js: true do
+    before :all do
+      set_valid_omniauth
+      OmniAuth.config.test_mode = true
+      Category.create({name: "Alex", description: "Alex's special category for crappy things"})
```

Comment:
Line is too long. [95/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.